bx_django_utils/feature_flags/admin_views.py

```diff
@@ -84,6 +84,11 @@ def form_valid(self, form):
             feature_flag.set_state(new_state)
             change_message = f'Set "{feature_flag.human_name}" to {feature_flag.state.name}'
 
+            if hasattr(feature_flag, '_cache_duration'):
+                now = feature_flag._cache_time_func()
+                remaining_duration = feature_flag._cache_duration.total_seconds() - (now - feature_flag._cache_from)
```

Comment:
This is only correct for the process handling this request, right?

We should just use `remaining_duration = feature_flag._cache_duration.total_seconds()`, because that's the maximum duration it takes, right?

Reply:
You're right. Fixed.